aurorastation.dme

```diff
@@ -1,3278 +1,3283 @@
-// DM Environment file for aurorastation.dme.
```

Comment:
why so many line changes in the DME?

Reply:
I believe it's because I fecked up merging master at one point, I've gone through and I _think_ nothing got removed when I fixed it